src/pages/Inputs/InputDemo.jsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console */
+import React from 'react';
+
+import { selectOptions, radioOptionsCricket, radioOptionsFootball } from '../../config/constants';
+import { TextField, SelectField, RadioField } from '../../components';
+
+
+class InputDemo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      sport: '',
+      cricket: '',
+      football: '',
+    };
+  }
+
+handleNameChange = (e) => {
+  this.setState({ name: e.target.value }, () => console.log(this.state));
+}
+
+handleSportChange = (e) => {
+  this.setState({ sport: e.target.value }, () => console.log(this.state));
```

Comment:
e.target.value === 'cricket' ? this.setState({ football: '' }) : this.setState({ cricket: '' })
try this in 'handleSportChange' instead of if else.

Reply:
try same for handlePositionChange